flowsa/data/flowbyactivitymethods/Census_AHS.yaml

```diff
@@ -1,21 +1,20 @@
-# Downloads and parses CSV from US Census American Housing Survey (AHS) at national level
----
 author: US Census Bureau
 source_name_bib: American Housing Survey
 citable_url: https://www.census.gov/programs-surveys/ahs.html
-date_generated: "2020-03-11" # updated automatically when flowbyactivity.py is run
+date_generated: '2021-04-28' # updated automatically when flowbyactivity.py is run
 bib_id: Census_AHS
 api_name: Census_AHS
 api_key_required: false
 format: csv
 url:
 #  base_url: "https://www2.census.gov/programs-surveys/ahs/__year__/AHS __year__ National PUF __ver__ Flat CSV.zip"
-  base_url: "https://www2.census.gov/programs-surveys/ahs/__year__/AHS __year__ National PUF __ver__ CSV.zip"
+  base_url: https://www2.census.gov/programs-surveys/ahs/__year__/AHS __year__ National
```

Comment:
this url is giving me errors, I think due to the spaces

2021-05-18 12:21:00 INFO     Calling https://www2.census.gov/programs-surveys/ahs/2017/AHS 2017 National PUF v3.0 CSV.zip
2021-05-18 12:22:01 ERROR    Error in URL request!
BadZipFile: File is not a zip file

When I enter into a browser it auto updates to https://www2.census.gov/programs-surveys/ahs/2017/AHS%202017%20National%20PUF%20v3.0%20CSV.zip

Reply:
@catherinebirney did you see this comment from before?

Reply:
@bl-young Missed this comment. Census_AHS FBA was never finished because we used a published statistic instead of creating a statistic using the database. So I should delete the .py/.yaml for this data source.